static/js/components/search/FilterVisibilityToggle.js

```diff
@@ -71,6 +72,20 @@ export default class FilterVisibilityToggle extends SearchkitComponent {
     />;
   };
 
+  renderFilterName = (children: React$Element<*>): React$Element<*>|null => {
+    if (!this.isInResults(children.props.id)) {
+      return null;
+    }
+    const { title } = this.props;
+    return (
+      <div
+        className="sk-hierarchical-refinement-list__header"
+        onClick={this.toggleFilterVisibility}>
+        {title}
+      </div>
+    );
+  };
```

Comment:
why not roll this into the `openStateIcon` method above? these two methods basically do the same thing (check `this.isInResults` and if so return some UI, else null).

You could write a new method like this:

```js
renderFilterTitle = (children) => {
  if (!this.isInResults(children.props.id)) {
    return null;
  }
  const { title } = this.props;
  return <div className="title-row" onClick={this.toggleFilterVisibility}>
    <Icon
      name="arrow_drop_down"
      className={this.openClass()}
    />
    <div className="sk-hierarchical-refinement-list__header">       <div className="sk-hierarchical-refinement-list__header">
      { title } 
    </div>
  </div>;
}
```

something along those lines! then we're just making the check once, only setting one `onClick` handler, etc.
             
what do you think?